cli/Main.hs

```diff
@@ -89,3 +108,9 @@ optsP = (,,)
     noTabularP = O.flag' (mkOptionsMorphism $ \opts -> opts { optTabular = False })
         $ O.long "no-tabular"
 
+    checkFormattedP = O.flag' (mkOptionsMorphism $ \opts -> opts { optCheckFormatted = True })
+        $ O.long "check-formatted" <> O.help "Fail with non-zero exit code if input is not formatted"
+
+    noCheckFormattedP = O.flag' (mkOptionsMorphism $ \opts -> opts { optCheckFormatted = False })
+        $ O.long "no-check-formatted"
```

Comment:
and this `--format` with appropriate help messages pointing out that these are two mode of operations (we might have third, though I have no idea what that could be :)

Reply:
In that case, do you think we should disable output for the check case? It could actually be three mutually-exclusive modes:

 * print to stdout
 * update in-place
 * check

Reply:
Yes

Reply:
Done.